News_App/src/app/app.module.ts

```diff
@@ -18,8 +20,18 @@ import { HttpClientModule } from '@angular/common/http';
     FooterComponent,
     FormComponent,
     FilterPipe,
+    AuthenticationComponent,
+  ],
+  imports: [
+    BrowserModule,
+    AppRoutingModule,
+    FormsModule,
+    HttpClientModule,
+    AuthModule.forRoot({
+      domain: 'news-ku.us.auth0.com',
```

Comment:
You should move this credentials in environment files

Reply:
rescheduled